src/App.js

```diff
@@ -1,42 +1,45 @@
-import React, { useEffect } from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddtodoForm';
-import "./style.css"
+import React from 'react';
+import TodoList from './TodoList.js';
+import AddTodoForm from './AddtodoForm.js';
+import './style.css';
 
-function useSemiPersistentState(){
-
-  let savedTodoList = localStorage.getItem("savedTodoList");
+function App() {
+  let savedTodoList = localStorage.getItem('savedTodoList');
   let [todoList, setTodoList] = React.useState(JSON.parse(savedTodoList || '[]'));
+  let [isLoading, setIsLoading] = React.useState(true);
```

Comment:
It is best to use `const` instead of 'let' when you will not be reassigning a variable.

Reply:
Thanks for the tip!